package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>sophus</name>
+  <version>1.0.0</version>
+  <description>
+   C++ implementation of Lie Groups using Eigen.
+  </description>
+  <url type="repository">https://github.com/stonier/sophus</url>
+  <url type="bugtracker">https://github.com/stonier/sophus/issues</url>
+  <maintainer email="nomail@nomail.com">Hauke Strasdat</maintainer>
+  <author>Hauke Strasdat</author>
+  <license>BSD</license>
+
+  <buildtool_depend>cmake</buildtool_depend>
+
+  <build_depend>eigen</build_depend>
+
+  <exec_depend>catkin</exec_depend>
```

Comment:
What is the reason for an exec_depend on catkin? I think it can be removed, no?

Reply:
Ok, maybe let's just leave it in, but it seem like it is only relevant if you release it into a ROS distro with bloom and then build debian pacakges, see the comment on why this was originally added to the example you linked to: https://github.com/ros-infrastructure/rep/pull/23#discussion_r3228465.